doc/tutorial/conv_arithmetic.txt

```diff
@@ -878,8 +878,69 @@ Here is an example for :math:`i = 6`, :math:`k = 3`, :math:`s = 2` and :math:`p
 .. figure:: conv_arithmetic_figures/padding_strides_odd_transposed.*
     :figclass: align-center
 
+Miscellaneous convolutions
+==========================
+
+Dilated convolutions
+--------------------
+
+Those familiar with the deep learning literature may have noticed the term
+"dilated convolutions" (or *convolutions à trous*) appear in recent papers. Here
```

Comment:
There is also the term "atrous convolution", the english version. Add it somewhere to help people find it. They could grep for that.